c_src/quicer_ctx.h

```diff
@@ -93,11 +95,13 @@ typedef struct QuicerConnCTX
   ErlNifMonitor owner_mon;
   ErlNifEnv *env;
   ErlNifMutex *lock;
+#if defined(QUICER_USE_TRUSTED_STORE)
   X509_STORE *trusted;
-  // Connection handle closed flag
-  // false means the handle is invalid
+#endif // !QUICER_USE_TRUSTED_STORE
```

Comment:
Note: the comment says `!QUICER_USE_TRUSTED_STORE`, but the condition is triggered when it's defined.

Reply:
Good catch, I removed the #else ... part but leave this behind....